include/mxnet/c_api.h

```diff
@@ -895,6 +940,10 @@ MXNET_DLL int MXNDArrayGetAuxType(NDArrayHandle handle,
                                   mx_uint i,
                                   int *out_type);
 
+MXNET_DLL int MXNDArrayGetAuxTypeInt64(NDArrayHandle handle,
+                                       mx_int64 i,
+                                       int *out_type);
```

Comment:
shouldn't out_type be mx_int64* ?

Reply:
not sure whether it should be int64 or not. out_type gets the type of aux data